### PR TITLE
Re-sending offer emails

### PR DESCRIPTION
**Pull request by osamaramihafez**
When an offer is re-sent, the offer should indicate any differences between it and the last offer sent.

### CODE REVIEW

File: backend/app/mailers/offer_mailer.rb [hunk 1]
```diff
@@ -38,6 +38,14 @@ def email_nag(offer)
 
     private
 
+    def get_prev_offer_diff
+        id = @offer.assignment_id
+        latest =
+            Offer.where('assignment_id = ?', id).order('emailed_date DESC')
+                .limit(1)
+        return @offer.compute_diff(latest[0])
```
Comment (Accelsnow) [inline backend/app/mailers/offer_mailer.rb]: I think you can omit the 'return' on this line

File: backend/app/views/offer_mailer/email_contract.html.erb [hunk 2]
```diff
@@ -6,11 +6,19 @@
     </style>
   </head>
   <body>
+  <p><%= @status_message %></p>
+  <br>
   <p>Dear <%= @first_name %> <%= @last_name %></p>
   <p>
     We would like to offer you a position as a TA for <%= @position_code %> <%= @position_title %>.
   </p>
-  <p>
```
Comment (siefkenj) [inline backend/app/views/offer_mailer/email_contract.html.erb:13]: Why did you move this to inside if? Whitespace doesn't matter at the start of a `<p>`. Also, are the tags closed correctly?
